DependencyInjection/NorthernMaintenanceModeExtension.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Northern\MaintenanceModeBundle\DependencyInjection;
+
+class NorthernMaintenanceModeExtension extends \Symfony\Component\HttpKernel\DependencyInjection\Extension
+{
+    /**
+     * @inheritDoc
+     */
+    public function load(array $configs, \Symfony\Component\DependencyInjection\ContainerBuilder $container)
```

Comment:
For bundles, I'd like to not use FQN in order to be consistent with the community.

Reply:
Updated here and in the rest of the MR